templates/company/filing_history/view_content.html.tx

```diff
@@ -97,34 +97,44 @@
     </table>
 
     % if $company_filing_history.total_count > $entries_per_page {
-            <ul class="pager">
-                % if ($previous_page) {
-                    <li>
-                        <a id="previousButton" class="page" href="<% $c.url_with.query([ page => $previous_page ]) %>" data-page="<% $previous_page %>">
-                            Previous
-                        </a>
-                    </li>
-                % }
+        <nav class="govuk-!-margin-top-6 govuk-!-margin-bottom-2 govuk-pagination" aria-label="Pagination">
```

Comment:
We need to review if the `govuk-!-margin-*` classes are appropriate here, since the spacing we had on the other results page may not transfer to this page.

Are there screenshots of how the pagination looks below the table?